backend/typings/mgr_module.pyi

```diff
@@ -1,12 +1,20 @@
 import ceph_module
 import logging
-import rados
+import rados as radosImport
```

Comment:
out of curiosity why the alias `rados` -> `radosImport`?

Reply:
pyright was not picking up that import due to reasons, and I was creative. :)

Reply:
Would pyright also be okay with naming it `rados_import`, possibly?

Reply:
nevermind, I've addressed in a different way by importing solely the Rados class and using that instead.